.github/workflows/_unittest.yml

```diff
@@ -66,4 +67,5 @@ jobs:
         # Run pytest with coverage
         ${CONDA_RUN} pytest -n auto --cov=./ --cov-report=xml
         # Run gtest
+        LLVM_PROFDATA="xcrun llvm-profdata" LLVM_COV="xcrun llvm-cov" \
```

Comment:
Our MacOS runner doesn't have xcode, so this won't work for now. But that's probably ok for now because running unit test on Linux returns the coverage fine

Reply:
I see. Strange - seems that xcrun is working though

Reply:
Oh sorry I just see it's my local xcrun works, not on server